python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -195,7 +195,15 @@ def test_invalid_connection_strings(self):
         for i in invalid:
             self.assertRaises(AttributeError, ChannelBuilder, i)
 
-        self.assertRaises(AttributeError, ChannelBuilder("sc://host/;token=123").to_channel)
+    def test_sensible_defaults(self):
+        chan = ChannelBuilder("sc://host")
```

Comment:
we already discussed but just as a reminder .. would better change `sc` to others ... like `connect` or `http` .. anything else (in a separate PR).

Reply:
yes, i will change it to "http" makes the parsing above easier as well, but needs a separate PR because it needs to update the dev doc as well.

Reply:
I might has missed your previous discussion:

why using, for example, `http` which is not a true http protocol in the underlying implementation?

Reply:
GRPC uses http2 so HTTP is most likely the most relevant schema but so could be h2. 

The reason to pick http is that it aligns nicely with what customers and users might expect.

Reply:
sounds good then